### DISCUSSION TITLE
Run tests on a matrix of Go versions on Linux/macOS/Windows

### DISCUSSION THREAD

**Discussion by qba73** [tufanbarisyildirim/gonginx#33] [Ideas]
Hi,
Before commencing any work I want to ask if maintainers are okay with the idea of updating the github action to run tests on 3 latest Go versions (v1.21, 1.20, 1.19) and 3 operating systems: Linux, MacOS, Windows.

**Comment by tufanbarisyildirim** [comment]
hah, quite funny and I'm sorry I missed this message:)
